<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinCodeUI</class>
 <widget class="QDialog" name="PinCodeUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>257</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>PIN code window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="pinCodeTextBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ForbiddenCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="maxLength">
          <number>4</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Enter PIN code</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="infoTextBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="1">
        <widget class="QPushButton" name="btn_0">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="btn_9">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>9</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btn_6">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btn_3">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_1">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_4">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_5">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btn_7">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btn_8">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="btn_ok">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btn_clear">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btn_cancel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
